doc/reference/configuration/configuration_reference.rst

```diff
@@ -1874,6 +1872,365 @@ instances
+    *   ``syslog``: write logs to a system logger (see :ref:`log.syslog.* <configuration_reference_log_syslog>`).
+
+    |
+    | Type: string
+    | Default: 'stderr'
+    | Environment variable: TT_LOG_TO
+
+
+..  _configuration_reference_log_file:
+
+..  confval:: log.file
+
+    Specify a file for logs destination.
+    To write logs to a file, you need to set :ref:`log.to <configuration_reference_log_to>` to ``file``.
+    Otherwise, ``log.file`` is ignored.
+
+    **Example**
+
+    The example below shows how to write logs to a file placed in the specified directory:
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/log_file/config.yaml
+        :language: yaml
+        :start-at: log:
+        :end-at: instance.log
+        :dedent:
+
+    Example on GitHub: `log_file <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/log_file>`_.
+
+    |
+    | Type: string
+    | Default: 'var/log/{{ instance_name }}/tarantool.log'
+    | Environment variable: TT_LOG_FILE
+
+
+..  _configuration_reference_log_format:
+
+..  confval:: log.format
+
+    Specify a format is used for a log entry.
+    The following formats are supported:
+
+    *   ``plain``: a log entry is formatted as plain text. Example:
+
+        ..  code-block:: text
+
+            2024-04-09 11:00:10.369 [12089] main/104/interactive I> log level 5 (INFO)
+
+    *   ``json``: a log entry is formatted as JSON and includes additional fields. Example:
+
+        ..  code-block:: text
+
+            {
+              "time": "2024-04-09T11:00:57.174+0300",
+              "level": "INFO",
+              "message": "log level 5 (INFO)",
+              "pid": 12160,
+              "cord_name": "main",
+              "fiber_id": 104,
+              "fiber_name": "interactive",
+              "file": "src/main.cc",
+              "line": 498
+            }
+
+    |
+    | Type: string
+    | Default: 'plain'
+    | Environment variable: TT_LOG_FORMAT
+
+
+..  _configuration_reference_log_level:
+
+..  confval:: log.level
+
+    Specify the level of detail logs have.
+    There are the following levels:
+
+    * 0 -- ``fatal``
+    * 1 -- ``syserror``
+    * 2 -- ``error``
+    * 3 -- ``crit``
+    * 4 -- ``warn``
+    * 5 -- ``info``
+    * 6 -- ``verbose``
+    * 7 -- ``debug``
+
+    By setting ``log.level``, you can enable logging of all events with severities above or equal to the given level.
+
+    **Example**
+
+    The example below shows how to log all events with severities above or equal to the ``VERBOSE`` level.
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/log_level/config.yaml
+        :language: yaml
+        :start-at: log:
+        :end-at: verbose
+        :dedent:
+
+    Example on GitHub: `log_level <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/log_level>`_.
+
+    |
+    | Type: number, string
```

Comment:
The option here accepts both string and number as values, as stated in [instance_config.lua](https://github.com/tarantool/tarantool/blob/60fdffbb6c2850e6d4a8436cec6162a595d863d1/src/box/lua/config/instance_config.lua#L800)

In the [box.cfg reference](https://docs.d.tarantool.io/en/doc/logging/reference/configuration/#confval-log_level), only the number is mentioned - is this correct?

Reply:
> The option here accepts both string and number as values, as stated in [instance_config.lua](https://github.com/tarantool/tarantool/blob/60fdffbb6c2850e6d4a8436cec6162a595d863d1/src/box/lua/config/instance_config.lua#L800)
> 
> In the [box.cfg reference](https://docs.d.tarantool.io/en/doc/logging/reference/configuration/#confval-log_level), only the number is mentioned - is this correct?

Checked - also accepts string values. Also fixed string values that `log_level` accepts. For example, `CRITICAL` is an incorrect value. Thanks!